### PR TITLE
dev: test ansible-core bump in toolbox build

### PR DESCRIPTION
**Pull request by cloudguruab**
My initial assumption would be that this would fail simply because we arent updating other parts of the ci-matrix, and build scripts inside the scripts folder? Let me know the idea behind only changing container env for ansible-core, and not other portions of code base to support this.

### CODE REVIEW

**Comment by jistr** [comment]
For the import errors we probably want to look at `import openstack` in these files:

```
os_migrate/plugins/module_utils/project.py:import openstack
os_migrate/plugins/module_utils/router.py:import openstack
os_migrate/plugins/module_utils/server_port.py:import openstack
os_migrate/plugins/module_utils/image.py:import openstack
os_migrate/plugins/module_utils/network.py:import openstack
os_migrate/plugins/module_utils/server.py:import openstack
os_migrate/plugins/module_utils/subnet.py:import openstack
os_migrate/plugins/module_utils/security_group_rule.py:import openstack
os_migrate/plugins/module_utils/keypair.py:import openstack
os_migrate/plugins/module_utils/reference.py:import openstack
os_migrate/plugins/module_utils/flavor.py:import openstack
os_migrate/plugins/module_utils/user.py:import openstack
os_migrate/plugins/module_utils/server_floating_ip.py:import openstack
os_migrate/plugins/module_utils/security_group.py:import openstack
os_migrate/plugins/module_utils/router_interface.py:import openstack
os_migrate/plugins/module_utils/server_volume.py:import openstack
```

and solve the import as suggested at:

https://docs.ansible.com/ansible/latest/dev_guide/testing/sanity/import.html

**Comment by cloudguruab** [comment]
https://opendev.org/openstack/ansible-collections-openstack/src/branch/master/docs/openstack_guidelines.rst 
https://docs.ansible.com/ansible/2.10/dev_guide/platforms/openstack_guidelines.html
No luck on debugging changes with resource files, and two other files as well. Found some documentation online that could be helpful to review during our sync. 
https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_general.html#creating-a-module
Could write a custom module, but I dont think this is needed, nor a general solution for this issue.

File: os_migrate/plugins/module_utils/image.py [hunk 1]
```diff
@@ -2,12 +2,18 @@
 __metaclass__ = type
 
 import hashlib
-import openstack
 import os
 
 from ansible_collections.os_migrate.os_migrate.plugins.module_utils \
     import const, exc, reference, resource
 
+try:
```
Comment (jistr) [inline os_migrate/plugins/module_utils/image.py:10]: Looks like we can drop the import indirection changes

Comment (jistr) [reply os_migrate/plugins/module_utils/image.py:10]: And we need the "mode" line additions from https://github.com/os-migrate/os-migrate/pull/523/files

**Comment by cloudguruab** [comment]
Messed up on rebase :/

**Comment by cloudguruab** [comment]
@jistr need your approval on this to merge

**Pull request merged**